projects/rocprofiler-systems/source/lib/core/agent.hpp

```diff
@@ -55,6 +55,8 @@ struct agent
 
     size_t device_type_index{ 0 };
     size_t base_id{ 0 };
+
+    std::string agent_info;
```

Comment:
Maybe a comment here:

```suggestion
    std::string agent_info; // JSON-serialization of agent information
```